projects/iio_demo/src.mk

```diff
@@ -86,23 +88,22 @@ LSCRIPT = $(PROJECT)/src/linker/STM32F446RETX_FLASH.ld
 endif
 
 ifeq (linux,$(PLATFORM))
+CFLAGS += -DENABLE_IIO_NETWORK \
+			-DDISABLE_SECURE_SOCKET
 
-SRC_DIRS += $(NO-OS)/libraries/mbedtls/include/mbedtls
+LIBRARIES += iio
 SRCS += $(NO-OS)/network/linux_socket/linux_socket.c 
 SRCS +=	$(NO-OS)/network/tcp_socket.c
 SRCS += $(PROJECT)/src/app/main.c
 SRCS += $(NO-OS)/iio/iio_app/iio_app.c
-SRCS	 += $(NO-OS)/util/circular_buffer.c
+SRCS += $(NO-OS)/util/circular_buffer.c
 
 SRCS += $(DRIVERS)/platform/generic/uart.c \
 		$(DRIVERS)/platform/generic/delay.c
 
 INCS += $(NO-OS)/network/tcp_socket.h \
 		$(NO-OS)/network/network_interface.h \
 		$(NO-OS)/network/noos_mbedtls_config.h \
-		$(NO-OS)/network/wifi/at_parser.h \
-		$(NO-OS)/network/wifi/wifi.h \
-		$(NO-OS)/network/wifi/at_params.h \
```

Comment:
aducm3029 may use this files:
Can you do?
ifeq ($(PLATFORM), linux)
   IGNORED_FILES = $(NO-OS)/network/wifi
endif

Reply:
I'm not sure I understand, the above changes are already only for the linux platform, the aducm3029 tincludes the whole network folder

Reply:
I see, my mistake